assets/src/components/detours/detourDropdown.tsx

```diff
@@ -54,6 +54,7 @@ export const DetourDropdown = ({
               zoom,
             })
           }}
+          data-fs-element="Add Detour on Trip"
```

Comment:
non-blocking: Perhaps, this instead, given we're not adding a detour for only one trip, but just via the vehicle entrypoint
```suggestion
          data-fs-element="Add Detour via Vehicle Marker"
```